src/presentation/components/Signup/Signup.tsx

```diff
@@ -0,0 +1,82 @@
+import { Button, Typography } from '@material-ui/core';
+import { InputText } from '../../../infrastructure/components/InputText/InputText';
+import { Modal } from '../../../infrastructure/components/Modal/Modal';
+import { useSignup } from './useSignup';
+import useTranslation from '../../../app/hooks/useTranslation';
+import { Link } from 'react-router-dom';
+import styles from './Signup.module.scss';
+
+export const Signup = () => {
+  const t = useTranslation();
+  const {
+    formValues,
+    signupModalIsOpen,
+    handleOpenSigninModal,
+    handleClose,
+    handleOnSubmit,
+    error,
+    handleOnChange,
+  } = useSignup();
+  const { email, password, passwordConfirmation } = formValues;
+
+  return (
+    <Modal isOpen={signupModalIsOpen} handleClose={handleClose}>
+      <form className={styles.signupForm} onSubmit={handleOnSubmit}>
+        <div className={styles.signupForm__title}>
+          <Typography variant="h5">{t('signup.title')}</Typography>
+        </div>
+
+        {error && (
+          <div className={styles.signupForm__error}>
+            <Typography variant="h6">{error}</Typography>
+          </div>
+        )}
+
+        <hr />
+
+        <InputText
+          className={styles.signupForm__input}
+          label={t('signup.email')}
+          name="email"
+          type="email"
+          value={email}
+          error={error ? true : false}
```

Comment:
in this case, there's no need to use a ternary operator, javascript will compute it into a truthy or falsy value, so you can just use `error={error}`. In case you need it to be boolean, you can do `error={!!error}`. Same on lines 53 and 63. What do you think?

Reply:
Great. I need `error` to be boolean so I will go with your second suggestion. Thanks